src/main/java/com/unstoppabledomains/resolution/Uttilities.java

```diff
@@ -0,0 +1,21 @@
+package com.unstoppabledomains.resolution;
+
+abstract class Utilities {
```

Comment:
`Utilities` usually references as a class with only static membrs. It is better to put these methods inside `NamingService` class.